cfme/tests/automate/test_domain.py

```diff
@@ -291,3 +294,68 @@ def test_object_attribute_type_in_automate_schedule(appliance):
         view.flash.assert_no_error()
         view.form.object_type.select_by_visible_text('<Choose>')
         view.flash.assert_no_error()
+
+
+@pytest.mark.tier(3)
+def test_copy_to_domain(domain):
+    """This test case checks whether automate class, instance and method are successfully copying to
+    domain.
+
+    Polarion:
+        assignee: ghubale
+        casecomponent: Automate
+        caseimportance: low
+        initialEstimate: 1/15h
+        startsin: 5.10
```

Comment:
Why change it to 5.10 from 5.9? I think if the feature was introduced in 5.9, it should be kept that way as an FYI, even if we will no longer test 5.9.

Reply:
Yeah agree! I know about the ```startsin```.  But as we are not supporting ```5.9```. So intentionally, It have changed to 5.10. Although that is not required field in polarion tags.